react/src/lib/components/DeckGLMap/storybook/DeckGLMap.stories.jsx

```diff
@@ -274,6 +274,7 @@ const meshMapLayer = {
     bounds: [432205, 6475078, 437720, 6481113],
     meshMaxError: 100,
     mesh: "hugin_depth_25_m_normalized_margin.png",
+    // terrainWorkerUrl: "terrain-worker.js",  //  KEEP. Optional. Will default to an inlined version of the script.
```

Comment:
Maybe better to create a separate story using the example terrain worker script.